src/components/vanilla/charts/TableChart/TableChart.emb.ts

```diff
@@ -60,26 +61,20 @@ export const meta: EmbeddedComponentMeta = {
     },
     {
       name: 'defaultSortDirection',
-      type: SortDirectionType,
+      type: SortDirectionType as never,
```

Comment:
will it also work without the never ? i can see our users forgetting this never

Reply:
No, we cannot type such custom types in SDK. There's always a workaround though, just cast entire `meta` to any if user doesn't want to deal with proper typing